alphagov/notifications-api
# Go live requester receipt

When users send service go live request, users don’t know where their request has gone or what to do if they don’t hear back. Send the request receipt email to the requester. More info on [this card](https://trello.com/c/POFY2Kdu/1067-implement-the-receipt-email-for-request-to-go-live-self-approve).

To manually test:
- Enable 'Can approve own go-live requests' from organisation settings
- Select trial service and request to go live from settings
- As a requester you should receive receipt email
